scholarx/2022/index.html

```diff
@@ -170,7 +170,7 @@ <h1 class="display-3 text-center mb-5">Program Timeline</h1>
             <div class="row justify-content-center">
                 <div class="col-11 text-center">
                     <p class="display-3 text-onelive font-weight-bolder mb-0">Let's hear it from our</p>
-                    <p class="display-3 text-onelive font-weight-bolder">ScholarX Alumini</p>
+                    <p class="display-3 text-onelive font-weight-bolder">ScholarX Alumni</p>
```

Comment:
Are you sure there are no other places with this mistake? @dilankavishka

Reply:
https://github.com/sef-global/sef-site/pull/1476#pullrequestreview-1219456498

Reply:
I'm corrected the other misspelling word also.

Reply:
https://github.com/sef-global/sef-site/pull/1476#pullrequestreview-1219456498 here you are not corrected it @dilankavishka

Reply:
There is a word `alumini` in the success-stories page @dilankavishka 

https://github.com/sef-global/sef-site/blob/bad025879edd857f31680f1c9b823974a5cb6b2f/scholarx/success-stories/index.html#L162

![image](https://user-images.githubusercontent.com/97069900/207903238-07971f92-d229-4350-b8e4-3fd749a66649.png)

Change it to `alumni`

Reply:
I changed it.

Reply:
But the changes are not committed @dilankavishka

Reply:
Check the preview link it's not corrected https://pr-1476-sef-site.surge.sh/scholarx/success-stories